config.toml

```diff
@@ -28,13 +28,11 @@ proj = ["x", "y", "z"]
 slice = []
```

Comment:
Random idea, maybe create a `config.toml.example` file with de default settings. Then let the user create a `config.toml` from this file. The `config.toml` will then be put in the gitignore. This way the local settings are not pushed every commit

Reply:
Great idea!

Reply:
I've opened an issue (#18) for this